src/instruments/src/RMP/Components/BaseRadioPanels.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { VhfRadioPanel } from './VhfRadioPanel';
+import { RadioPanelDisplay } from './RadioPanelDisplay';
+import { useSimVar, useInteractionSimVar } from '../../Common/simVars';
+import { useInteractionEvent } from '../../Common/hooks';
+
+/**
+ *
```

Comment:
These empty comments might fool the computer! But humans are less easily fooled! 🤣